hw/ip/otbn/data/enc-schemes.yml

```diff
@@ -253,15 +265,29 @@ bnan:
       bits: 19-15
       value: bxxxxx
 
-# Used for bn.mulmv
-bnammv:
+
+# bnammv below is used for new bn.mulv instruction
+bnmulv:
```

Comment:
It looks to me like `bnammv` gets deleted here and not added back, does this "below" reference actually make sense?